hoodie-client/src/main/java/com/uber/hoodie/config/HoodieWriteConfig.java

```diff
@@ -136,25 +143,24 @@ public String getWriteStatusClassName() {
     return props.getProperty(HOODIE_WRITE_STATUS_CLASS_PROP);
   }
 
-  public boolean shouldUseTempFolderForCopyOnWriteForCreate() {
-    return Boolean.parseBoolean(props.getProperty(HOODIE_COPYONWRITE_USE_TEMP_FOLDER_CREATE));
+  public int getFinalizeWriteParallelism() {
```

Comment:
Make this `Integer` instead of `int` like other getters in this class ?

Reply:
The older configs (getXXXShuffleParallelism) is returning primitives. I have changed all the newly added configs to follow the same convention.